_documentation/user/flags/wings.md

```diff
@@ -5,4 +5,25 @@ type: Good
 summary: Tank can drive in air.
 ---
 
-...
+Wings flag allows the player to maneuver their tank while in the air, enabling them to move forward, backward, or turn to either side (x and y axis), all while the gravity still affects the tank midair (z axis).
```

Comment:
"flapping" provides "negative gravity", or a force opposing gravity. The z-axis description is incomplete.